packages/docs/docs/communications/organizing-communications.md

```diff
@@ -0,0 +1,480 @@
+# Organizing Communications
+
+# Introduction
+
+Communications in a healthcare context (modeled as [Communication](/docs/api/fhir/resources/communication) resources) can include many scenarios – patient to physician, physician to physician, device to physician, and more. This results in a large volume of communications within this context, so ensuring that they are well organized is vital. For example, a provider may have staff on call to respond to patient messages. Without proper organization, messages may not reach the correct staff member.
+
+This guide covers different ways that you can organize communications and how to think about different properties on the `Communication` element. It will cover:
+
+- Common communication organization patterns, including `Communication.category`, `Communication.topic`, and threading.
+- Searching for communications and threads using the the search helper method from the [Medplum Client SDK](/docs/sdk/classes/MedplumClient)
+- Sorting communications by time
+- How to use different Codesystems to classify communications.
+- Querying for communications.
+
+## Communication Organization Patterns
+
+There are various ways to organize communications using FHIR, but the most common are `Communication.category` and `Communication.topic`. Additionally, you can use properties such as `Communication.inResponseTo` and `Communication.partOf` to organize communications using threading.
```

Comment:
> Additionally, you can use properties such as `Communication.inResponseTo` and `Communication.partOf` to organize communications using threading.

This seems redundant with the section lower down

This is a big question that we get from our customers, so we need to give a more detailed recommendation here. Some questions that we need to answer: 
1. What is the difference between using `inResponseTo` and `partOf`? IMO, `irt` points to the directly previous message, but `partOf` requires a separate Comms resource to model the whole thread
2. If we have a parent/child relationship of `Communication` resources using `partOf`, how do we distinguish a "thread-level" vs. "message level" `Communication`
3. When should we use a `Communication` as a top-level grouping mechanism, vs. an `Encounter` as we describe here: https://www.medplum.com/docs/communications/async-encounters